code_block_timer/storage.py

```diff
@@ -15,16 +15,15 @@
 
 import os
 import sqlite3
-from path import path
```

Comment:
I had added `path.py` to the requirements.txt file - I'll remove it from that file.